operations/services.md

```diff
@@ -1585,8 +1585,8 @@ Creates a new order with the specified products and items. Only positive charges
 | Property | Type | Contract | Description |
 | --- | --- | --- | --- |
 | `ProductId` | string | required | Unique identifier of the [Product](services.md#product) to be ordered. |
-| `Count` | number | optional | Count of products to be ordered, e.g. 10 in case of 10 beers. |
-| `UnitAmount` | [Amount](services.md#amount-parameters) | optional | Unit amount of the product that overrides the amount defined in Mews. |
+| `Count` | number | optional | Count of products to be ordered, e.g. 10 in case of 10 beers. The number sent must be a positive integer.|
```

Comment:
By doing this, we’re forcing partners to do rebates incorrectly. Proper rebate should have negative count and positive amount. @vitasamek please discuss with Josef Lapka, they’ve been doing this in commander and it isnt correct since after rebate, you’ll have 2 items in statistics, not zero.

Reply:
@siroky I reverted the change here https://github.com/MewsSystems/gitbook-connector-api/pull/275
I will discuss it. Thank you